control/xferfcn.py

```diff
@@ -758,7 +760,12 @@ def __pow__(self, other):
             return (TransferFunction([1], [1]) / self) * (self**(other + 1))
 
     def __getitem__(self, key):
+        if not isinstance(key, Iterable) or len(key) != 2:
+            raise IOError('must provide indices of length 2 for state space')
```

Comment:
Change "state space" to "transfer functions".

Reply:
I have updated in the latest commit.